specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBRestorableMongodbCollectionList.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid",
+    "location": "WestUS",
+    "instanceId": "98a570f2-63db-4117-91f0-366327b7b353",
+    "restorableMongodbDatabaseRid": "PD5DALigDgw="
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/providers/Microsoft.DocumentDb/locations/westus/restorableDatabaseAccounts/98a570f2-63db-4117-91f0-366327b7b353/restorableMongodbCollections/79609a98-3394-41f8-911f-cfab0c075c86",
+            "type": "Microsoft.DocumentDB/locations/restorableDatabaseAccounts/restorableMongodbCollections",
+            "name": "79609a98-3394-41f8-911f-cfab0c075c86",
+            "properties": {
+              "resource": {
+                "_rid": "zAyAPQAAAA==",
+                "eventTimestamp": "2020-10-13T04:56:42Z",
+                "ownerId": "Collection1",
+                "ownerResourceId": "V18LoLrv-qA=",
+                "operationType": "Create"
+              }
```

Comment:
same comment: Not returning the MongoDBCollection in here ??

Reply:
We are not including the content for now.  Please see the other comment for details.